src/components/Home.js

```diff
@@ -100,6 +102,20 @@ export class Home extends Component {
     return true
   }
 
+  handleRandomize() {
+    const { username } = this.props
+    fetch(`${this.props.url}/randomize`, {
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      method: 'post',
+      body: JSON.stringify({ username }),
+    }).then(response => response.json())
+      .then(({ message }) => ToastAndroid.show(message, ToastAndroid.SHORT))
+      .catch(err => ToastAndroid.show(err.toString(), ToastAndroid.SHORT))
+  }
+
```

Comment:
Well, this fetch call really should be elsewhere ^^.
Sending headers inside a component is meh.